--- a/src/qtbase.mk
+++ b/src/qtbase.mk
@@ -33,7 +33,10 @@
 $(PKG)_DEPS := zlib libpng freetype fontconfig sqlite libjpeg \
-              dbus libicu pcre2 harfbuzz xz libxml2 \
-              $(if $(filter $(MXE_WINDOWS_BUILD),y), \
-                   openssl libmysqlclient)
+              dbus libicu pcre2 harfbuzz xz libxml2 \
+              $(if $(filter $(MXE_WINDOWS_BUILD),y), \
+                   openssl)